test-packages/e2e-tests/test/bound-function.spec.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  TestEntity,
+  TestEntityWithMultipleKeys,
+  testService
+} from '@sap-cloud-sdk/test-services-e2e/v4/test-service';
+import { getByKey } from '@sap-cloud-sdk/test-services-e2e/v4/test-service/function-imports';
+
+const url = 'http://localhost:4004/';
+const destination = { url };
+
+describe('bound functions', () => {
+  describe('integer parameter, returns entity', () => {
+    const request = getByKey({
+      param: 101
+    });
+
+    it('bound function returns expected string', async () => {
+      // http://localhost:4004/odata/test-service/TestEntity(KeyTestEntity=1)/TestService.boundFunctionWithoutArguments()
+      const entity: TestEntity = await request.execute(destination);
+      const functionResult = await entity
+        .boundFunctionWithoutArguments({})
+        .execute(destination);
+      expect(functionResult).toEqual('xyz');
+    });
+
+    it('bound function of entity with multiple keys returns expected string', async () => {
+      // http://localhost:4004/odata/test-service/TestEntityWithMultipleKeys(KeyTestEntityWithMultipleKeys=101,StringPropertyWithMultipleKeys='a',BooleanPropertyWithMultipleKeys=true)/TestService.boundFunctionWithoutArgumentsWithMultipleKeys()
+      const { testEntityWithMultipleKeysApi } = testService();
+      const entity: TestEntityWithMultipleKeys =
+        await testEntityWithMultipleKeysApi
+          .requestBuilder()
+          .getByKey(101, 'a', true)
+          .execute(destination);
+
+      const functionResult = await entity
+        .boundFunctionWithoutArgumentsWithMultipleKeys({})
+        .execute(destination);
+      expect(functionResult).toEqual('xyz');
```

Comment:
[req] This returns the same value as the line 23, please use a different value.

Reply:
improved with a unique value